scout/server/blueprints/variant/templates/variant/acmg.html

```diff
@@ -105,6 +105,14 @@ <h4>Evidence of {{ category }}</h4>
           </div>
         {% endfor %}
       </div>
+    <!-- external links -->
+    <div class="card panel-default mt-3">
+      <div class="card-body">
+        {% for gene in variant.genes %}
+          <a href="{{ gene.cspec_link }}" class="btn btn-secondary text-white" rel="noopener" referrerpolicy="no-referrer" target="_blank">Search {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }} ClinGen CSPEC</a>
```

Comment:
```suggestion
          <a href="{{ gene.cspec_link }}" class="btn btn-secondary text-white" rel="noopener" referrerpolicy="no-referrer" target="_blank">Search {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }} in ClinGen CSPEC</a>
```

Reply:
Hm, right. It is also a bit long anyway. Perhaps we add a little text first, then just the gene buttons, or a tooltip with the long version!

Reply:
Like so?
![Screenshot 2025-03-28 at 15 29 25](https://github.com/user-attachments/assets/0438f3a2-1a88-4c19-ad36-c80a4a30a9fd)